crates/ruma-common/src/events/room/message/content_serde.rs

```diff
@@ -50,3 +52,52 @@ impl<'de> Deserialize<'de> for MessageType {
         })
     }
 }
+
+/// The payload for a file message.
+#[derive(Clone, Debug, Deserialize)]
+#[cfg(feature = "unstable-msc3551")]
+pub struct FileMessageEventContentSerDeHelper {
```

Comment:
A comment saying why we need this instead of just being able to use the `alias` attribute would be good.